core/controllers/base_test.py

```diff
@@ -717,15 +718,18 @@ def test_get(self):
 class LogoutPageTests(test_utils.GenericTestBase):
     """Tests for logout handler."""
 
-    def test_logout_page(self):
+    def test_logout_page_calls_destroy_auth_session(self):
```

Comment:
Don't we also need to verify that the session is created at the correct time, too?

Reply:
Yep, but that's the `SessionBeginHandler`'s responsibility which has its own (just improved) tests.